tests/generic_views/test_edit.py

```diff
@@ -81,6 +81,13 @@ def get_form(self, form_class):
             MissingDefaultValue().get_form(), forms.Form,
         )
 
+    def test_get_context_data(self):
+        class FormContext(FormMixin):
+            request = RequestFactory().get('/')
+            form_class = forms.Form
+
+        self.assertIn('form', FormContext().get_context_data())
```

Comment:
Maybe `self.assertIsInstance(FormContext().get_context_data()['form'], forms.Form)` would be slightly better?